pkg/kubelet/certificate/bootstrap/bootstrap.go

```diff
@@ -344,7 +344,8 @@ func requestNodeCertificate(ctx context.Context, client clientset.Interface, pri
 		return nil, err
 	}
 
-	reqName, reqUID, err := csr.RequestCertificate(client, csrData, name, certificatesv1.KubeAPIServerClientKubeletSignerName, usages, privateKey)
+	var duration *time.Duration // TODO: pick a good value for the CSR expirationSeconds field and/or allow it to be configured
```

Comment:
drop the todo, I'd expect a cluster-level kubelet signer duration rather than per-request config like this